flow360/component/geometry.py

```diff
@@ -97,9 +97,9 @@ def _validate_geometry(self):
             if not os.path.exists(geometry_file):
                 raise Flow360FileError(f"{geometry_file} not found.")
 
-        if self.name is None and len(self.file_names) > 1:
+        if self.project_name is None and len(self.file_names) > 1:
             raise Flow360ValueError(
-                "name field is required if more than one geometry files are provided."
+                "`project_name` field is required if more than one geometry files are provided."
```

Comment:
can we just use the name of first file provided? We can sent a warning

Reply:
Done